src/components/PrintDimension.tsx

```diff
@@ -0,0 +1,55 @@
+import { Slider, Switch } from "antd";
```

Comment:
Make sure to remove any unused import statements, there are a few on this page :)

Reply:
Yep good catch :)